chain/indexer/integrated/processor/state_internal_test.go

```diff
@@ -0,0 +1,112 @@
+package processor
+
+import (
+	"testing"
+
+	saminer1 "github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	saminer2 "github.com/filecoin-project/specs-actors/v2/actors/builtin/miner"
+	saminer3 "github.com/filecoin-project/specs-actors/v3/actors/builtin/miner"
+	saminer4 "github.com/filecoin-project/specs-actors/v4/actors/builtin/miner"
+	saminer5 "github.com/filecoin-project/specs-actors/v5/actors/builtin/miner"
+	saminer6 "github.com/filecoin-project/specs-actors/v6/actors/builtin/miner"
+	saminer7 "github.com/filecoin-project/specs-actors/v7/actors/builtin/miner"
+	"github.com/ipfs/go-cid"
+	"github.com/stretchr/testify/require"
+
+	init_ "github.com/filecoin-project/lily/chain/actors/builtin/init"
+	"github.com/filecoin-project/lily/chain/actors/builtin/market"
+	"github.com/filecoin-project/lily/chain/actors/builtin/miner"
+	"github.com/filecoin-project/lily/chain/actors/builtin/multisig"
+	"github.com/filecoin-project/lily/chain/actors/builtin/power"
+	"github.com/filecoin-project/lily/chain/actors/builtin/reward"
+	"github.com/filecoin-project/lily/chain/actors/builtin/verifreg"
+	"github.com/filecoin-project/lily/chain/indexer/tasktype"
+	"github.com/filecoin-project/lily/tasks/actorstate"
+	inittask "github.com/filecoin-project/lily/tasks/actorstate/init_"
+	markettask "github.com/filecoin-project/lily/tasks/actorstate/market"
+	minertask "github.com/filecoin-project/lily/tasks/actorstate/miner"
+	multisigtask "github.com/filecoin-project/lily/tasks/actorstate/multisig"
+	powertask "github.com/filecoin-project/lily/tasks/actorstate/power"
+	rawtask "github.com/filecoin-project/lily/tasks/actorstate/raw"
+	rewardtask "github.com/filecoin-project/lily/tasks/actorstate/reward"
+	verifregtask "github.com/filecoin-project/lily/tasks/actorstate/verifreg"
+	"github.com/filecoin-project/lily/tasks/blocks/drand"
+	"github.com/filecoin-project/lily/tasks/blocks/headers"
+	"github.com/filecoin-project/lily/tasks/blocks/parents"
+	"github.com/filecoin-project/lily/tasks/chaineconomics"
+	"github.com/filecoin-project/lily/tasks/consensus"
+	"github.com/filecoin-project/lily/tasks/messageexecutions/internalmessage"
+	"github.com/filecoin-project/lily/tasks/messageexecutions/internalparsedmessage"
+	"github.com/filecoin-project/lily/tasks/messages/blockmessage"
+	"github.com/filecoin-project/lily/tasks/messages/gaseconomy"
+	"github.com/filecoin-project/lily/tasks/messages/gasoutput"
+	"github.com/filecoin-project/lily/tasks/messages/message"
+	"github.com/filecoin-project/lily/tasks/messages/parsedmessage"
+	"github.com/filecoin-project/lily/tasks/messages/receipt"
+	"github.com/filecoin-project/lily/tasks/msapprovals"
+)
+
+func TestNewProcessor(t *testing.T) {
```

Comment:
ensure a new processor from all tasks has the expected processors registered. Will prevent issues like: https://github.com/filecoin-project/lily/pull/996